sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/ui/ThriftServerPage.scala

```diff
@@ -422,70 +335,25 @@ private[ui] class SessionStatsPagedTable(
 
   override def goButtonFormPath: String = {
     val encodedSortColumn = URLEncoder.encode(sortColumn, UTF_8.name())
-    s"$parameterPath&$sessionStatsTableTag.sort=$encodedSortColumn&$sessionStatsTableTag.desc=$desc"
+    s"$parameterPath&$sessionStatsTableTag.sort=$encodedSortColumn" +
+      s"&$sessionStatsTableTag.desc=$desc#$sessionStatsTableTag"
   }
 
   override def headers: Seq[Node] = {
-    val sessionTableHeaders =
-      Seq("User", "IP", "Session ID", "Start Time", "Finish Time", "Duration", "Total Execute")
-
-    val tooltips = Seq(None, None, None, None, None, Some(THRIFT_SESSION_DURATION),
-      Some(THRIFT_SESSION_TOTAL_EXECUTE))
-    assert(sessionTableHeaders.length == tooltips.length)
-    val colWidthAttr = s"${100.toDouble / sessionTableHeaders.size}%"
-
-    val headerRow: Seq[Node] = {
-      sessionTableHeaders.zip(tooltips).map { case (header, tooltip) =>
-        if (header == sortColumn) {
-          val headerLink = Unparsed(
-            parameterPath +
-              s"&$sessionStatsTableTag.sort=${URLEncoder.encode(header, UTF_8.name())}" +
-              s"&$sessionStatsTableTag.desc=${!desc}" +
-              s"&$sessionStatsTableTag.pageSize=$pageSize" +
-              s"#$sessionStatsTableTag")
-          val arrow = if (desc) "&#x25BE;" else "&#x25B4;" // UP or DOWN
-            <th width={colWidthAttr}>
```

Comment:
`PagedTable#headerRow` you added doesn't support the width of `th`.
Did you confirm it doesn't matter for `ThriftServerPage`?

Reply:
Yeah, its working fine.
<img width="1677" alt="Screenshot 2020-05-20 at 7 26 03 PM" src="https://user-images.githubusercontent.com/15366835/82455377-84328580-9ad0-11ea-855d-1947b5aa8485.png">